chcemvediet/apps/anonymization/migrations/0001_initial.py

```diff
@@ -16,7 +16,7 @@ class Migration(migrations.Migration):
             fields=[
                 ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
                 ('successful', models.BooleanField(default=False, help_text='True if normalization has succeeded, False otherwise.')),
-                ('file', models.FileField(help_text="NULL if normalization failed or normalization didn't create any file.", max_length=255, null=True, upload_to='attachment_normalizations')),
+                ('file', models.FileField(help_text="Empty filename if normalization failed or normalization didn't create any file.", max_length=255, null=True, upload_to='attachment_normalizations')),
```

Comment:
migracia nam nezodpoveda modelu, kontretne pole `file`, kde v migracii null=True nie je nahredene s blank=True

Reply:
@viliambalaz Vdaka, ze si si to vsimol. Pls sprav samostatny PR, kde to fixneme. Kedze sme este nemali release, tak migracie, co sme pridali teraz, mozeme este opravovat.

Pls, pre istotu pozri aj ostatne prigracie, co sme pridali, ci tam nie je nieco, co nesedi.